### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by lvarin**
Minor changes from the document in google docs.

### CODE REVIEW

File: 4.2-federated-framework/README.md [hunk 1]
```diff
@@ -3,24 +3,39 @@
...
-Main scope of this deliverable is to gather technical requirements & frameworks for federated analysis platform. In work package subtask 4.3.2, the project partners have described mainly following use cases: 
-* Federated QTL analysis for molecular phenotypes.
-* Simple Workflow Polygenic Risk Scores (PRS) across two similar ethnic background sample sets.
-* Federated joint cohort genotyping
 
-The federated analysis platform defined by this task aims in providing technological solutions for these use cases. Technical requirements are thus gathered based upon these use case descriptions. The aim of this deliverable is to write a short design document that shows the requirements and lists the different options for the solution. 
+The main scope of this deliverable is to gather technical requirements and frameworks for a federated analysis platform. In the work package subtask number 4.3.2, the project partners have described mainly the following use cases:
+
+- A federated QTL analysis for molecular phenotypes.
+- A simple Workflow for Polygenic Risk Scores (PRS) across two similar ethnic background sample sets.
```
Comment (tskir) [inline 4.2-federated-framework/README.md]:
```suggestion
- A simple workflow for Polygenic Risk Scores (PRS) across two similar ethnic background sample sets.
```

File: 4.2-federated-framework/README.md [hunk 2]
```diff
@@ -3,24 +3,39 @@
 Note: the deliverable text has since been updated [in Google Docs](https://docs.google.com/document/d/1YJI3Ya5CV7g_V-Vm3xeCF0pJqpcm37zm_pCvoZydH_I).
```
Comment (tskir) [inline 4.2-federated-framework/README.md]:
```suggestion
```

Now that this page is updated, we can make it the main one again

File: 4.2-federated-framework/README.md [hunk 3]
```diff
@@ -29,64 +44,236 @@ As a starting point for this deliverable, a Data Workflow Survey for work packag
 | **Data storage**  | EGA (controlled access), dbGap (controlled access), ENA (open access), GEO (open access)  |
 | **Transfer protocol**   |  http(s) (manual), (s)ftp (manual), Globus (manual), Aspera (manual), htsget (automated)  |
 |  **Access rights**|  Per-user basis (no external access), token based (ELIXIR AAI, Switch AAI)  |
-#### Figure 1: Data workflow
-![Survey Data Transfer](survey-data-transfer.png)
+
+### Figure 1: Data workflow
+
+![Survey Data Transfer](survey-data-transfer.drawio.png)
```
Comment (tskir) [inline 4.2-federated-framework/README.md:48]: It looks like there might be a problem with image paths: they are referenced directly from the `/4.2-federated-framework` folder, but are actually committed in `/docs/4.2-federated-framework`. I suggest moving them to `/4.2-federated-framework` for consistency

File: 4.2-federated-framework/README.md [hunk 4]
```diff
@@ -29,64 +44,236 @@ As a starting point for this deliverable, a Data Workflow Survey for work packag
...
+These 4 API standards are inspired by large-scale distributed compute projects and, in theory they could be implemented for different computing and data archive environments. The Figure 2 depicts the typical functional architecture of a computing ecosystem proposed by GA4GH Cloud Web Services.
+
+### Figure 2: GA4GH Compatible Cloud Platform (Functional Architecture)
+
+![ga4gh cloud](./ga4gh-cloud.png)
 
 ## Deployment Scenarios
 
-Following deployment scenarios for Federated Genomics analysis cloud APIs were discussed:
+8The following 3 deployment scenarios were discussed for Federated Genomics analysis cloud APIs.
```
Comment (tskir) [inline 4.2-federated-framework/README.md]:
```suggestion
The following 3 deployment scenarios were discussed for Federated Genomics analysis cloud APIs.
```

File: 4.2-federated-framework/README.md [hunk 5]
```diff
@@ -29,64 +44,236 @@ As a starting point for this deliverable, a Data Workflow Survey for work packag
...
-Under this deployment scenario, CINECA WP4 partners can deploy a [Nextflow](https://www.nextflow.io/) manager (& Optionally a WES shim to make it fully compatible with GA4GH WES) which serve as WES endpoint. Nextflow executor in this scenario is [TESK](https://github.com/EMBL-EBI-TSI/TESK) which act as a TES endpoint. For data access, it is assumed data is made available to data staging area within the cluster. Figure 4, depicts this deployment model where a centralized WES (Nextflow manager) & federated TES (TESK) endpoints are deployed CINECA WP4 wide partners.
+
+Under this deployment scenario, CINECA WP4 partners can deploy a [Nextflow](https://www.nextflow.io/) manager (and Optionally a WES shim to make it fully compatible with GA4GH WES) which serve as a WES endpoint. Nextflow executor in this scenario is [TESK](https://github.com/EMBL-EBI-TSI/TESK) which act as a TES endpoint. For data access, it is assumed the data is made available to the data staging area within the cluster. Figure 4, depicts this deployment model where a centralized WES (Nextflow manager) and federated TES (TESK) endpoints are deployed in several CINECA WP4 wide partners.
```
Comment (tskir) [inline 4.2-federated-framework/README.md]:
```suggestion
Under this deployment scenario, CINECA WP4 partners can deploy a [Nextflow](https://www.nextflow.io/) manager (and optionally a WES shim to make it fully compatible with GA4GH WES) which serve as a WES endpoint. Nextflow executor in this scenario is [TESK](https://github.com/EMBL-EBI-TSI/TESK) which act as a TES endpoint. For data access, it is assumed the data is made available to the data staging area within the cluster. Figure 4, depicts this deployment model where a centralized WES (Nextflow manager) and federated TES (TESK) endpoints are deployed in several CINECA WP4 wide partners.
```

File: 4.2-federated-framework/README.md [hunk 6]
```diff
@@ -29,64 +44,236 @@ As a starting point for this deliverable, a Data Workflow Survey for work packag
...
+- The Nextflow manager delegates processes to the corresponding TES endpoints.
+- The private analysis is executed on the orresponding TES endpoints.
+- The results of the analysis from different TES endpoints are then aggregated at a specific TES endpoint or WES endpoint.
+- The final aggregated result is made available to the end user.
+
+### dbGap: Data Access Workflow
+
+![dbGAP](./dbGAP.drawio.png)
+
+**Prerequisites**: THe SRA toolkit should be installed in the Data Staging Area, the User must have access to the access controlled data and have access to its dbGAP repository key in the Data staging area.
```
Comment (tskir) [inline 4.2-federated-framework/README.md]:
```suggestion
**Prerequisites**: The SRA toolkit should be installed in the Data Staging Area, the User must have access to the access controlled data and have access to its dbGAP repository key in the Data staging area.
```

**Review by tskir — COMMENTED** [review]
Great work @lvarin, thanks for doing this! I've left a few suggestions, most notably including the image paths—I think they need to be moved to a different folder

File: 4.2-federated-framework/README.md (hunk 3 above)
Comment (lvarin) [reply 4.2-federated-framework/README.md:48]: Moved

**Comment by lvarin** [comment]
All changes applied

**Review by tskir — APPROVED** [review]
Everything's great now!

**Pull request merged**